tinygrad/tensor.py

```diff
@@ -632,7 +632,7 @@ def floor(self: Tensor) -> Tensor: return (self < (b := self.trunc())).where(b-1
   def square(self): return self*self
   def clip(self, min_, max_): return self.maximum(min_).minimum(max_)
   def abs(self): return self.relu() + (-self).relu()
-  def sign(self): return self / (self.abs() + 1e-10)
+  def sign(self): return ((self.float()) / (self.float().abs() + 1e-12)).cast(self.dtype)
```

Comment:
Oh, I meant the other way, with the comparisons. The dividing is terrible

Reply:
i know i know - I tried it but test_ops breaks because we can't do a backward pass on a less than node

Reply:
https://github.com/tinygrad/tinygrad/actions/runs/7058043827/job/19212838796#step:10:176 test run logs

Reply:
I'm not sure sign is supposed to be differentiable, is torch's?

Reply:
it's not https://discuss.pytorch.org/t/torch-sign-breaks-autograd-dynamic-graph/90711/2 - but you _can_ compute the grad (will be 0.0)